### PR TITLE
Feature/magic8ball

### PR DESCRIPTION
**Pull request by rhydiaan**
Gromit, that's it! Cheese!

### CODE REVIEW

File: judge/main.py [hunk 1]
```diff
@@ -95,6 +95,21 @@ async def coinflip(ctx, arg=None):
...
+    """
+    Asks the heavens for an answer
+    Usage:
+        tellme [Your question]
+        tellme help
+    """
+    try:
+        await bot.tell_me_main(ctx, arg)
+    except Exception as e:
+        logger.error('Failed to handle Coinflip:')
```
Comment (SamTipper) [inline judge/main.py:110]: This should be "Failed to handle 8 Ball" or something

**Review by SamTipper — CHANGES_REQUESTED** [review]

File: judge/main.py (hunk 1 above)
Comment (rhydiaan) [reply judge/main.py]: Fixed the logger error, should be fine for master now

**Review by SamTipper — APPROVED** [review]
Looks good boss!

**Comment by Coombszy** [comment]
Don't forget to merge 😉 I got you this time.

**Pull request merged**